### DISCUSSION TITLE
Proper Sudo Checks

### DISCUSSION THREAD

**Discussion by kphanipavan** [spacedriveapp/spacedrive#174] [Feature Request]
![Screenshot_20220523_101613](https://user-images.githubusercontent.com/60005847/169745238-db74e8cd-21a5-4b50-8d9a-2420f4a5b1f3.png)

Running the script in a container, which defaults to root user, fails to setup because sudo doesn't exist. Having sudo check just like cargo and pnpm helps setup the work environment in a container.

Using Docker Container with lateset Ubuntu image.

**Comment by oscartbeaumont** [comment]
I was using a sudo check but it was causing issues (at least for me) because Rust or Node (can't remember which) is installed in a user context not machine context so running `sudo ./scriptnamehere` would cause it to be run as the root user. I will look into this when I get some time.
